eclass/coreos-go-utils.eclass

```diff
@@ -87,7 +87,13 @@ go_export() {
 	# Remove certain flags from $LDFLAGS to fix validation errors in
 	# Go >= 1.15.5 like:
 	#   go build runtime/cgo: invalid flag in go:cgo_ldflag: -Wl,-O1
-	filter-ldflags "-Wl,-O1"
+	#
+	# Note, we need to manually strip only the optimization element of
+	# comma-separated flags, e.g. from `-Wl,-O1,-s` to `-Wl,-s`.
+	# To support multiple characters that can follow `-O`, e.g. `-Ofast`,
+	# we should use regexp like `[[:alnum:]]*`.
```

Comment:
Maybe add a comment that this is a work-around for https://github.com/golang/go/pull/42631

Reply:
Added the comment